inventory/sample/group_vars/k8s-cluster/k8s-net-flannel.yml

```diff
@@ -14,3 +14,8 @@
 # for experimental backend
 # please refer to flannel's docs : https://github.com/coreos/flannel/blob/master/README.md
 # flannel_backend_type: "vxlan"
+# flannel_vxlan_vni: 1
+# flannel_vxlan_port: 8472
+# For Windows Container, use following values:
```

Comment:
Might be nice to support host-gateway flannel deployment for windows nodes.

Reply:
It has already supported by followng configuration
```
flannel_backend_type: "host-gw"
```